packages/@lwc/engine-server/src/__tests__/fixtures/adjacent-text-nodes/empty/modules/x/comments-text/comments-text.html

```diff
@@ -0,0 +1,3 @@
+<template>
+    <span>{foo}{bar}{baz}</span>
```

Comment:
```suggestion
    <!-- should render a single zero-width joiner (U+200D) -->
    <span>{foo}{bar}{baz}</span>
```
help out the dummies 😅

Reply:
This won't end up in the `expected.html` so it doesn't really help.

I considered using `lwc:preserve-comments` or adding a `<div>` or something but I felt this would make things less clear.

Reply:
```suggestion
    The span should contain a single zerow-width joiner (U+200D).
    <span>{foo}{bar}{baz}</span>
```